lib/MC/WinCOFFStreamer.cpp

```diff
@@ -200,9 +200,13 @@ void MCWinCOFFStreamer::EmitCOFFSectionIndex(MCSymbol const *Symbol) {
 }
 
 void MCWinCOFFStreamer::EmitCOFFSecRel32(MCSymbol const *Symbol) {
-  MCDataFragment *DF = getOrCreateDataFragment();
   const MCSymbolRefExpr *SRE = MCSymbolRefExpr::create(Symbol, getContext());
-  MCFixup Fixup = MCFixup::create(DF->getContents().size(), SRE, FK_SecRel_4);
+  EmitCOFFSecRel32Value(SRE);
+}
+
+void MCWinCOFFStreamer::EmitCOFFSecRel32Value(MCExpr const *expr) {
```

Comment:
expr should be capitalized per coding convention.
Make sure something like "git diff -U0 HEAD^ | c:\Python27\python.exe  F:\LLVM-REL35\llvm\tools\clang\tools\clang-format\clang-format-diff.py -i -p1"